packages/design/src/DataTable/Table.tsx

```diff
@@ -56,30 +59,51 @@ export function Table<T>({
   const renderBody = (data: T[]) => {
     const rows = [];
 
-    data.map((item, rowIdx) => {
-      const cells = columns.map((column, columnIdx) => {
-        const $cell = column.render ? (
-          column.render(item)
-        ) : (
-          <TextCell data={item[column.key]} />
-        );
+    if (fetching?.fetchStatus === 'loading') {
+      return <LoadingIndicator colSpan={columns.length} />;
+    } else {
+      data.map((item, rowIdx) => {
+        const cells = columns.map((column, columnIdx) => {
+          const $cell = column.render ? (
```

Comment:
Does the `$` prefix on the variable name mean something special?

Reply:
It doesn't do anything special, it's just a naming convention to indicate that the variable contains a DOM node